decentraland/unity-renderer
# fix: chat entry pooling

## What does this PR change?

Adds object pooling for chat entries. Improves hiccups and performance.
Deleted obsolete prefabs in resource folders.

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/chat-message-pooling
2. Open different chat windows: nearby, channels, private conversations and check that every message is displayed correctly
3. Write messages in each window
4. Hover the mouse over messages, date should be displayed correctly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
